metadata-models/src/main/pegasus/com/linkedin/ml/metadata/CaveatsAndRecommendations.pdl

```diff
@@ -0,0 +1,17 @@
+namespace com.linkedin.ml.metadata
+
+/**
+ * This section should list additional concerns that were not covered in the previous sections. For example, did the results suggest any further testing? Were there any relevant groups that were not represented in the evaluation dataset? Are there additional recommendations for model use?
+ */
+record CaveatsAndRecommendations {
+
+  /**
+   * This section should list additional concerns that were not covered in the previous sections. For example, did the results suggest any further testing? Were there any relevant groups that were not represented in the evaluation dataset?
+   */
+  caveats: optional string
+
+  /**
+   * Recommendations on where this MLModel should be used.
+   */
+  recommendations: optional string
+}
```

Comment:
can add another field (as per the paper) to capture ideal characteristics of an evaluation dataset for this model?